libs/frontend/feature-build/src/components/component-menu/ComponentInputRow.vue

```diff
@@ -106,6 +107,7 @@ const datalist = computed<IDatalistOption[] | undefined>(() => {
 .value-preview {
   @mixin truncate;
   @mixin text 14px;
+  flex-grow: 1;
```

Comment:
This is to fix an issue when input value is so short that buttons are not pushed to the end/right.

![Screenshot 2023-11-23 at 3 57 29 PM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/5eb0e2a0-b2cc-4967-bf9d-06691f88cc98)

Reply:
Thanks! I've been trying to simplify the layout/css here and there, and I missed that case.